Chap_Terms.tex

```diff
@@ -105,9 +96,9 @@ \section{Naming Conventions}
 \item The \code{pmix_} prefix is used to denote function pointer and type definitions.
 \end{itemize}
 
-Users should not use the \textbf{\code{PMIX}}, \textbf{\code{PMIx}}, or \textbf{\code{pmix}} prefixes in their applications or libraries so as to avoid symbol conflicts with current and later versions of the \ac{PMIx} standard and implementations such as the \ac{PRI}.
-
-%%%%%%%%%%%
+Users should not use the \textbf{\code{PMIX}}, \textbf{\code{PMIx}}, or \textbf{\code{pmix}} prefixes in their applications or libraries so as to avoid symbol conflicts with \ac{PMIx} implementations.
```

Comment:
Two notes from the reading:
```suggestion
Users shall not use the \textbf{\code{PMIX}}, \textbf{\code{PMIx}}, or \textbf{\code{pmix}} prefixes in their applications or libraries so as to avoid symbol conflicts with \ac{PMIx} implementations.
```
(Note from the Reading in 2Q 2020 ASC Meeting)